### DISCUSSION TITLE
收集：开源布道师相关技能（或工具）的使用

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/open-source-best-practice#138] [Open Source Cases]
作为一名开源布道师，可能会通过如下的方式来传播技术、理念：

* 博客文章、教程、业界动态
* 视频录制
  * obs
* 视频录制（剪辑）
  * [openshot](https://github.com/OpenShot/openshot-qt) written by Python
  * [shotcut](https://github.com/mltframework/shotcut) written by C++
  * [olive](https://github.com/olive-editor/olive) written by C++
  * [kdenlive](https://github.com/KDE/kdenlive) written by C++ (Linux only)
  * [blender](https://github.com/blender/blender) written By C
* 音频录制（剪辑）
  * Audacity
* 图片编辑
  * Inkscape
  * [Pencil2D](https://www.pencil2d.org/)
  * [GIMP](https://github.com/GNOME/gimp)

**Comment by LinuxSuRen** [comment]
视频教程大纲：
* 如何利用 OBS 录制视频教程
  * 如何录制多音轨
  * 如何解决多音轨视频播放时只有一个声音的问题
* 如何解决视频录制过程中的噪音
* 对视频做简单的剪辑
  * 裁剪
  * 缩短视频长度（操作等待时间）
  * 标题模板（复用视频片头、片尾）
* 音频处理技巧（audacity）
  * 从视频中提取音频
  * 降噪处理
  * 降低（提高）区域声音
  * 区域静音处理
  * 填充静音区
  * 声音重复
  * 口癖处理
  * 快捷键设置
  * 导出时间线

**Comment by LinuxSuRen** [comment]
音频提取工具（免费、开源）：
* VLC
  * `File -> Convert/Stream`
* Quick Time Player
  * `File -> Export As -> Audio Only`

**Comment by LinuxSuRen** [comment]
播客录制：
* [播客制作指南 - bibiEdit](https://www.bilibili.com/video/BV1FR4y1j7bW)

**Comment by LinuxSuRen** [comment]
开源海报生成工具 https://github.com/Ovilia/acasia-poster

**Comment by LinuxSuRen** [comment]
* https://ossinsight.io/
* https://open-leaderboard.x-lab.info/

**Comment by shinyzhu** [comment]
在录制音频的时候，除了使用麦克风录制我们说话，一般会需要录制系统或程序的声音，比如录制一段播放器的音乐、会议软件的音频，这些音频本来就在电脑里，所以这个功能叫做“内录”。

此处推荐一款适合macOS的纯软件内录实现：[BlackHole](https://github.com/ExistentialAudio/BlackHole)，代码库里有详细的使用介绍，有需要的可以取用。

**Comment by LinuxSuRen** [comment]
Some tips about audio file handling.

1. Keep all the tracks separate
1. Noise reduction for all tracks
1. Edit the labels during the audio editing process
1. Normalize all the tracks

> You could use Audicatiy or other tools.
